validation-test/SIL/parse_stdlib.sil

```diff
@@ -1,5 +1,5 @@
 // RUN: rm -f %t.*
-// RUN: %target-sil-opt -enable-sil-verify-all=true -sil-disable-ast-dump %platform-module-dir/Swift.swiftmodule -module-name=Swift -o %t.sil
+// RUN: %target-sil-opt -enable-sil-verify-all=true -sil-disable-ast-dump $(if [[ -d %platform-module-dir/Swift.swiftmodule ]]; then echo %platform-module-dir/Swift.swiftmodule/%target-swiftmodule-name; else echo %platform-module-dir/Swift.swiftmodule; fi) -module-name=Swift -o %t.sil
```

Comment:
Can we split this up please?  This syntax is not supported on Windows, and this will cause a regression on the windows target.